src/TorchSharp/NN/Module.cs

```diff
@@ -142,12 +143,66 @@ protected virtual void Dispose(bool disposing)
                     }
                 }
 
+                [DllImport("LibTorchSharp")]
+                static extern void THSNN_Module_to_device_dtype(HType module, sbyte dtype, long deviceType, long deviceIndex);
+
                 [DllImport("LibTorchSharp")]
                 static extern void THSNN_Module_to_device(HType module, long deviceType, long deviceIndex);
 
                 [DllImport("LibTorchSharp")]
                 static extern void THSNN_Module_to_dtype(HType module, sbyte dtype);
 
+                public virtual Module to(Device device, ScalarType dtype)
```

Comment:
Haven't I seen this exact method body before. Can it be reused?

Reply:
The ScriptModule version is an override that calls to a different native method than non-script modules do.

Reply:
... but there are still opportunities for sharing. Thanks for pointing that out.

Reply:
Right, I think there is no need to override this method in SxriptModule.